Sources/MapboxMaps/Location/Puck/PuckType.swift

```diff
@@ -43,6 +49,33 @@ public struct Puck2DConfiguration: Equatable {
         self.shadowImage = shadowImage
         self.scale = scale
         self.showsAccuracyRing = showsAccuracyRing
+        self.accuracyRingColor = UIColor(red: 0.537, green: 0.812, blue: 0.941, alpha: 0.3)
+        self.accuracyRingBorderColor = UIColor(red: 0.537, green: 0.812, blue: 0.941, alpha: 0.3)
```

Comment:
Can we declare colors as contants?

Reply:
Default values have to be public. So the constant would also have to be public. I'd prefer a bit of repetition over adding a constant to the public API.

Reply:
Moving forward with merge. Let's revisit use of constants async with the rest of the team.